composer.json

```diff
@@ -62,5 +62,7 @@
                 "Tenancy": "Hyn\\Tenancy\\Facades\\TenancyFacade"
             }
         }
-    }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true
```

Comment:
@mikebronner Thank you for this PR. Do we have a package that needs dev stability?

Reply:
Great find @bkintanar this should be removed now that the dev dependency to Laravel was removed 👍